jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/SNMP.java

```diff
@@ -127,6 +127,21 @@ public void setStatus(int status) {
    public static String snmpTimestampFromIso(String isoTimestamp) throws ParseException {
       ZonedDateTime zdt = DateTimeUtils.isoDateTime(isoTimestamp);
 
+      
+      StringBuilder sb = new StringBuilder();
+
+      sb.append(String.format("%04X", zdt.getYear()));
+      sb.append(String.format("%02X", zdt.getMonthValue()));
+      sb.append(String.format("%02X", zdt.getDayOfMonth()));
+      sb.append(String.format("%02X", zdt.getHour()));
+      sb.append(String.format("%02X", zdt.getMinute()));
+      sb.append(String.format("%02X", zdt.getSecond()));
+      sb.append(String.format("%02X", zdt.getNano()));
+      return sb.toString();
+   }
+
+   public static String fourOneSnmpTimestampFromIso(String isoTimestamp) throws ParseException {

```

Comment:
What's the difference between this `fourOneSnmpTimestampFromIso` method and the `snmpTimestampFromIso` method above?

Reply:
The fourOneSnmpTimestampFromIso method doesn't parse seconds/milliseconds. That extra information isn't required in the FOURDOT1 spec but is required in 1218